components/prism-bbj.js

```diff
@@ -6,11 +6,11 @@
 			greedy: true
 		},
 		'string': {
-			pattern: /"(?:""|[!#$%&'()*,\/:;<=>?^\w +\-.])*"/,
+			pattern: /(['"])(?:(?!\1|\\).|\\.)*\1/,
```

Comment:
In the tests, `"foo "" bar"` is highlighted as 2 strings. Is that intentional?

Reply:
@RunDevelopment That's intentional